docs/usage/shoot-operations/secretbinding-to-credentialsbinding-migration.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SecretBinding to CredentialsBinding Migration
+---
+
+# SecretBinding to CredentialsBinding Migration
+
+With the introduction of the [`CredentialsBinding`](../../api-reference/security.md#credentialsbinding) resource a new way of referencing credentials through the `Shoot` was created.
+While `SecretBinding`s can only reference `Secret`s, `CredentialsBinding`s can also reference `WorkloadIdentity`s which provide an alternative authentication method.
+`WorkloadIdentity`s do not directly contain credentials but are rather a representation of the workload that is going to access the user's account.
+
+As `CredentialsBinding`s cover the functionality of `SecretBinding`s, the latter are considered legacy and will be deprecated in the future.
+This incurs the need for migration from `SecretBinding` to `CredentialsBinding` resources.
+
+> [!NOTE] Mind that the migration will be allowed only if the old `SecretBinding` and the new `CredentialsBinding` refer to the same exact `Secret`.
```

Comment:
```suggestion
> [!NOTE] 
> Mind that the migration will be allowed only if the old `SecretBinding` and the new `CredentialsBinding` refer to the same exact `Secret`.
```

I think this is required, otherwise the format is messed up (see preview).

Reply:
You are right, should have checked.